spread.yaml

```diff
@@ -61,19 +67,19 @@ prepare: |
     fi
     go get ./cmd/spread
 
-    # Enable IPv4 in LXD.
-    if [ -z "$(source /etc/default/lxd-bridge && echo $LXD_IPV4_ADDR)" ]; then
-        cat >> /etc/default/lxd-bridge <<END
-    LXD_IPV4_ADDR="10.0.8.1"
-    LXD_IPV4_NETMASK="255.255.255.0"
-    LXD_IPV4_NETWORK="10.0.8.1/24"
-    LXD_IPV4_DHCP_RANGE="10.0.8.2,10.0.8.254"
-    LXD_IPV4_DHCP_MAX="250"
-    LXD_IPV4_NAT="true"
-    END
-        systemctl stop lxd lxd-bridge
-        systemctl start lxd lxd-bridge
+    # Cache is only available if SEND_CACHE=1 is set, follow the steps:
+    # . Create dest dir: > mkdir tests/cache/lxd-ubuntu-16.04
+    # . Get image id: > lxc image list
+    # . Export image: > lxc image export <IAMGE-ID> tests/cache/lxd-ubuntu-16.04
+    # . Extract matadata: > cd tests/cache/lxd-ubuntu-16.04 && tar -xvf <MATADATA-FILE>
```

Comment:
s/mata/meta/

Can we please also have "-" as the list item prefix instead of "."

Also, why that tar in the last command?  Seems surprising that `lxc image import` cannot import what `lxc image export` exports.

Reply:
Agree, it is super weird but this is a lxd known issue and I found the solution in a forum.